tests/foreman/api/test_contentmanagement.py

```diff
@@ -659,26 +644,19 @@ def test_positive_iso_library_sync(self, capsule_configured):
     @pytest.mark.tier4
     @pytest.mark.skip_if_not_set('capsule', 'clients', 'fake_manifest')
     def test_positive_on_demand_sync(self, capsule_configured, default_sat):
-        """Create a repository with 'on_demand' sync, add it to lifecycle
-        environment with a capsule, sync repository, examine existing packages
-        on capsule, download any package, examine packages once more
+        """Create a repository with 'on_demand' policy, add it to a CV,
+        promote to an 'on_demand' Capsule's LCE, check artifacts were created,
+        download a published package, assert it matches the source.
 
         :id: ba470269-a7ad-4181-bc7c-8e17a177ca20
 
-        :expectedresults:
-
-            1. After initial syncing only symlinks are present on both
-               satellite and capsule, no real packages were fetched.
-            2. All the symlinks are pointing to non-existent files.
-            3. Attempt to download package is successful
-            4. Downloaded package checksum matches checksum of the source
-               package
+        :expectedresults: multiple assertions across the code pass
```

Comment:
This is too general.  If would be best to include what expectedresults are here so anyone reading this would quickly know.

Reply:
@vsedmik These steps and expected results are to help a reader (think new component owner) understand what the test automation in the function is doing, and checking. It also serves as a guide if for some reason the test needs to be manually stepped through.

We certainly don't expect you to repeat the automation assertions word for word, and expected results should actually align with the `:steps:` token, which was missing here.

Please update the expectedresults that were here to describe what you're now asserting in the test.